cmd/utils.go

```diff
@@ -138,13 +129,44 @@ var generateDocsCmd = &cobra.Command{
 	},
 }
 
+// completionCmd generates auto completion commands
+var completionCmd = &cobra.Command{
+	Use:       "completion <shell>",
+	Hidden:    true,
+	ValidArgs: []string{"zsh", "bash"},
+	Args:      cobra.MinimumNArgs(1),
+	Short:     "Generates auto completion scripts",
+	Long: `To load completion add the following to your ~/.bashrc or ~/.zshrc file
+
+. <(corectl completion zsh)
```

Comment:
This requires corectl to be in PATH, right? Do we need a note on that in README as a requirement?